pkg/maps/ctmap/ctmap.go

```diff
@@ -249,19 +259,20 @@ func (m *Map) DumpEntries() (string, error) {
 
 // NewMap creates a new CT map of the specified type with the specified name.
 func NewMap(mapName string, mapType MapType) *Map {
```

Comment:
While at it, I think you could unexport this function (maybe in a separate commit). It is only used within the `ctmap` package.

Reply:
Done.